### ISSUE TITLE
Keyboard navigation doesn't work as expected

### ISSUE THREAD

**Issue by pedrowindisch** [codesandbox/codesandbox-client#4138]
# 🐛 bug report

## Description of the problem
The keyboard navigation isn't working as expected. The first focused element just appears after pressing the `tab` key two times, and between the third and eighth presses, it just doesn't focus any element at all.

There's also a bug after focusing the `Create a sandbox` button, as shown in the GIF below.

![codesandbox-fixissue](https://user-images.githubusercontent.com/30203228/82112283-cb5ff580-9721-11ea-9c2e-d5ddd489ff0a.gif)

## How has this issue affected you? What are you trying to accomplish?
The issue hasn't really affected me a lot, but if someone with a motor disability who relies on the keyboard tries to use the app, it'd be highly difficult for them to do so.

The color of the button is very similar to its `outline` color, which might represent a problem for users who experience low vision conditions who want to navigate through the landing page.

### Your Environment

| Software         | Name/Version |
| Browser          | Chrome, 81
| Operating System | Windows 10
